backend/llm_implementation/rag.py

```diff
@@ -1,11 +1,15 @@
 from dotenv import load_dotenv
 from llama_index.core import VectorStoreIndex, SimpleDirectoryReader, StorageContext, load_index_from_storage
 import os.path
+from llm_implementation import prompt
 
-def analyze(file_name):
-    load_dotenv()
+rag_models = {}
```

Comment:
This gets reset every time the server is restarted

Reply:
for now i don't think that should matter. eventually we'll put this in a database anyway.